bin/apisix

```diff
@@ -38,8 +38,8 @@ local pkg_path_org = package.path
 
 local apisix_home = "/usr/local/apisix"
 local pkg_cpath = apisix_home .. "/deps/lib64/lua/5.1/?.so;"
-                  .. apisix_home .. "/deps/lib/lua/5.1/?.so;;"
-local pkg_path  = apisix_home ..    "/deps/share/lua/5.1/?.lua;;"
+        .. apisix_home .. "/deps/lib/lua/5.1/?.so;;"
```

Comment:
Please do not make changes that are not related to this PR, it will make the review difficult